apiary.apib

```diff
@@ -1196,6 +1251,15 @@ A business asset type group. Used to classify a business asset type and apply ba
 + `value`: General Maintenance (string, required) - The description of the purpose.
 + `isActive`: true (boolean, required) - Indicates if the purpose is active.
 
+## Trade Create (object)
++ `name`: Electrician (string, required) - The name of the trade.
++ `isActive`: true (boolean, required) - Indicates if the trade is active.
+
+## Trade (Base Audit Object)
```

Comment:
We need to start following the standards that @moniquejones08 came up with for the data structures.
https://readme.dudesoln.com/display/EA/Apiary+Documentation+Standards